ballista/scheduler/src/metrics/prometheus.rs

```diff
@@ -46,17 +46,14 @@ pub struct PrometheusMetricsCollector {
     completed: Counter,
     submitted: Counter,
     pending_queue_size: Gauge,
-    active_jobs: Gauge,
 }
 
 impl PrometheusMetricsCollector {
     pub fn new(registry: &Registry) -> Result<Self> {
         let execution_time = register_histogram_with_registry!(
             "job_exec_time_seconds",
             "Histogram of successful job execution time in seconds",
-            vec![
```

Comment:
I think we want to keep this change. We need to setup a way to make this configurable upstream but for now we want different buckets for our specific use case